clusterloader2/cmd/clusterloader.go

```diff
@@ -79,6 +80,7 @@ func initClusterFlags() {
 	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.KubemarkRootKubeConfigPath, "kubemark-root-kubeconfig", "KUBEMARK_ROOT_KUBECONFIG", "",
 		"Path the to kubemark root kubeconfig file, i.e. kubeconfig of the cluster where kubemark cluster is run. Ignored if provider != kubemark")
 	flags.BoolEnvVar(&clusterLoaderConfig.ClusterConfig.APIServerPprofByClientEnabled, "apiserver-pprof-by-client-enabled", "APISERVER_PPROF_BY_CLIENT_ENABLED", true, "Whether apiserver pprof endpoint can be accessed by Kubernetes client.")
+	flags.StringEnvVar(&clusterLoaderConfig.ClusterConfig.LoggingLevel, "logging-level", "LOGGING_LEVEL", "2", "0: errors and warnings, 1: info, 2: detailed info.")
```

Comment:
I'm against introducing this flag. klog already has the --v flag that you can use for this purpose

Reply:
Thanks,  I'll remove it and just add V() logging control.  @wojtek-t

Reply:
Any idea why the test failed? Thanks.